service.clocknotify/service.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: cp1252 -*-
```

Comment:
Probably irrelevant (I don't believe Kodis python interpreter parses this info)... Kodi uses UTF-8 not windows encoding.